src/content/structured/get-started/figma.mdx

```diff
@@ -1,57 +1,58 @@
-
+ 
+subTitle: "Reusable and flexible Figma components used to design, prototype and hand-off designs for apps and digital products."
+ 
 contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/get-started/figma.mdx"
 ---
-
-import { IcAlert } from "@ukic/react";
+ 
 import figmaFig1 from "./images/assets2x.png";
 import figmaFig2 from "./images/libraries2x.png";
-
+ 
 ## About the Figma UI Kit
-
+ 
 We've built a Figma UI Kit to use alongside the guidance and code you'll find on this website. It'll make it easier to prototype and build things quickly.
-
-All Figma resources have been built with our users in mind, with a strong focus on making sure accessibility is embedded from the start. That way what we can provide an amazing experience, usable for everyone
-
-## Accessing the library
-
-<IcAlert
-  title="Need to know"
-  message="The Figma UI Kit is currently only available to internal users, but we will be sharing it publicly soon."
-  variant="info"
-/>
-
-The UI Kit is turned on by default for all projects within our Figma organisation.
-
+ 
+All Figma resources have been built with our users in mind, with a strong focus on making sure accessibility is embedded from the start. That way what we can provide an amazing experience, usable for everyone.
+ 
+## Accessing the library for internal users
+ 
+The UI Kit is turned on by default for all projects within our community. Check that you are using the latest version.
+ 
 1. In a new or existing Figma design file, click 'Assets' from the left-hand side panel then click the 'Team library' icon (the book).
-
+ 
 <DoubleDoDontCaution>
-  <DoDontCaution
+<DoDontCaution
     imageSrc={figmaFig1}
     imageAlt="Image of the left-hand side panel in Figma with the 'Assets' tab open and icon buttons on the right of the panel including the 'Team library' button."
     state="none"
   />
 </DoubleDoDontCaution>
-
+ 
 2. Make sure the Intelligence Community Design System library is turned on.
-
+ 
 <DoubleDoDontCaution>
-  <DoDontCaution
+<DoDontCaution
     imageSrc={figmaFig2}
     imageAlt="Image of the Libraries menu in Figma with the Intelligence Community Design System library turned on."
     state="none"
   />
 </DoubleDoDontCaution>
-
-## How to use the Figma UI Kit components
-
-Find the details on the internal system to get started with Figma.
+ 
+## Not part of the Intelligence Community?
+ 
+So that everyone can use our resources, we've published our [Figma UI Kit]() to the Figma Community. 
+
+You can make a copy of it in your local files and use the components in your designs. This is a static only version and you won't automatically receive component updates. Check Figma Community regularly for new updates and versions.
+ 
+	1. Navigate to the [UK Intelligence Community UI Kit on Figma Community]().
+	2. Duplicate the library into your drafts and connect your project files. 
```

Comment:
Should a screenshot be added so that it can be more visual?

Reply:
Good idea.